var/spack/repos/builtin/packages/r-evaluate/package.py

```diff
@@ -31,9 +31,11 @@ class REvaluate(RPackage):
     """Parsing and evaluation tools that make it easy to recreate the command
     line behaviour of R."""
 
-    homepage = "https://github.com/hadley/evaluate"
-    url      = "https://cran.rstudio.com/src/contrib/evaluate_0.9.tar.gz"
+    homepage = "https://cran.r-project.org/package=evaluate"
+    url      = "https://cran.rstudio.com/src/contrib/evaluate_0.10.1.tar.gz"
+    list_url = homepage
```

Comment:
I think you may want to set `list_url` to `https://cran.r-project.org/src/contrib/Archive/evaluate/`. When I run `spack versions r-evaluate` using this new URL I don't pick up any of the archived versions.

Reply:
Most of the r packages works using list_url=homepage, I don't know why this only gives the newest version. I will change it to the archive url instead, and this works I think.